### PR TITLE
C# interface concept enhancement. Issue # 4894

### LINKED ISSUE #4894: [Edit] C#: Interfaces
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `Interfaces` concept entry under C#. The entry lives in `content/c-sharp/concepts/interfaces/interfaces.md`.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Edit the existing code block to demonstrate default implementations for some of the Interface members.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by LewisMK**
### Description

Editing the existing code block to demonstrate default implementations for some of the Interface members. Default implementations are now supported from C# version 8.0. 

### Issue Solved

Closes #4894  

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by cigar-galaxy82 — COMMENTED** [review]
Hi, I have suggested a change please modify

File: content/c-sharp/concepts/interfaces/interfaces.md [hunk 1]
```diff
@@ -51,61 +51,159 @@ class MyClass : MyInterface
...
+    {
+        Console.WriteLine("The cat only moves when there is food in his bowl.");
+    }
+    // Uses default implementations for MakeSound and Move
 }
 
-// Create a main method to test the program
-public class Program
+```
+## Codebyte
```
Comment (cigar-galaxy82) [inline content/c-sharp/concepts/interfaces/interfaces.md:131]: give a blank line between the heading and the codeblock

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @LewisMK thank you for contributing to Codecademy docs. I have suggested few changes please modify

**Comment by LewisMK** [comment]
Hi @cigar-galaxy82 , thank you for the feedback. Working on the changes.

**Comment by LewisMK** [comment]
Hi @cigar-galaxy82 I have made the requested changes. Please check and let me know if there is anything more I can do to make it better. Thank you.

**Review by cigar-galaxy82 — APPROVED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @LewisMK the changes look good its ready for next review

**Review by Sriparno08 — APPROVED** [review]
Looks good, @LewisMK! 👍

**Comment by LewisMK** [comment]
Thank you @cigar-galaxy82  and @Sriparno08 for the guidance and review.

**Pull request merged**